pypeit/scripts/identify.py

```diff
@@ -88,9 +88,14 @@ def main(args):
         arccen, arc_maskslit = wavecal.extract_arcs(slitIDs=[args.slit])
 
         # Launch the identify window
+        # TODO -- REMOVE THIS HACK
+        try:
+            nonlinear_counts = msarc.detector.nonlinear_counts()
+        except:
+            nonlinear_counts = 1e10
```

Comment:
To save editing the 1e10 in both places of this code, maybe change this to `None`?